src/private/CABasicAnimation+MotionAnimator.m

```diff
@@ -225,4 +220,17 @@ void MDMConfigureAnimation(CABasicAnimation *animation,
       springAnimation.initialVelocity = absoluteInitialVelocity / displacement;
     }
   }
+
+  // Update the animation's duration to match the proposed settling duration.
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wpartial-availability"
+  if ([animation isKindOfClass:[CASpringAnimation class]]) {
+    CASpringAnimation *springAnimation = (CASpringAnimation *)animation;
```

Comment:
Is referencing the class before it was introduced OK? I believe CASpringAnimation was introduced in iOS 9?
https://developer.apple.com/documentation/quartzcore/caspringanimation

Reply:
Not strictly an encouraged behavior, no. But this allows us to expand our min SDK support to iOS 8. Is a trade-off, but a relatively safe one.

Reply:
Ok. As long as it doesn't crash with undefined symbols (or generate compiler errors). Sounds good to me.